R/runClusterSummaryMetrics.R

```diff
@@ -7,6 +7,7 @@
 #' @param featureNames A string or vector of strings with each gene to aggregate.
 #' @param displayName A string that is the name of the column used for genes.
 #' @param groupNames The name of a colData entry that can be used as groupNames.
+#' @param scale Option to scale the data
```

Comment:
I would potentially also update all of these with default values. 
e.g. #' @param scale Option to scale the data. Default /code{FALSE}. Selected assay will not be scaled.